uber/models.py

```diff
@@ -689,7 +689,7 @@ def assign_badges(self, group, new_badge_count, new_badge_type=c.ATTENDEE_BADGE,
 
             if diff > 0:
                 for i in range(diff):
-                    group.attendees.append(Attendee(badge_type=new_badge_type, ribbon=ribbon_to_use, paid=c.PAID_BY_GROUP, **extra_create_args))
+                    group.attendees.append(Attendee(badge_type=new_badge_type, ribbon=ribbon_to_use, paid=paid, **extra_create_args))
```

Comment:
This fix allows us to click "This person isn't coming" on the Group Members page and have the blank badge end up with the same ``paid`` value as the badge being unset.